src/mainviews/create-job.tsx

```diff
@@ -495,6 +515,57 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
             environmentList={environmentList}
             value={props.model.environment}
           />
+          <TextField
+            label={trans.__('Maximum Retry Attempts')}
+            variant="outlined"
+            onChange={e => {
+              // Validate name
+              setErrors({
+                ...errors,
+                maxRetryAttempts: MaxRetryAttemptsError(e.target.value, trans)
+              });
+              handleNumericInputChange(e as ChangeEvent<HTMLInputElement>);
+            }}
+            error={!!errors['maxRetryAttempts']}
+            helperText={errors['maxRetryAttempts'] ?? ''}
+            value={props.model.maxRetryAttempts}
+            id={`${formPrefix}maxRetryAttempts`}
+            name="maxRetryAttempts"
+          />
+          <TextField
+            label={trans.__('Max Run Time (In Seconds)')}
```

Comment:
```suggestion
            label={trans.__('Maximum Run Time (seconds)')}
```

"Maximum" was not abbreviated above